.github/workflows/java_nightly.yml

```diff
@@ -64,6 +68,31 @@ jobs:
           fi
           echo $PREFIX
           archery crossbow download-artifacts -f java-jars -t binaries  $PREFIX
+      - name: Cache Repo
+        uses: actions/cache@v3
+        with:
+          path: repo
+          key: java-nightly-${{ github.run_id }}
+          restore-keys: java-nightly
+      - name: Sync from Remote
+        uses: ./arrow/.github/actions/sync-nightlies
+        with:
+          switches: -avzh --update --delete --progress
+          local_path: repo
+          remote_path: ${{ secrets.NIGHTLIES_RSYNC_PATH }}/arrow/java
+          remote_host: ${{ secrets.NIGHTLIES_RSYNC_HOST }}
+          remote_port: ${{ secrets.NIGHTLIES_RSYNC_PORT }}
+          remote_user: ${{ secrets.NIGHTLIES_RSYNC_USER }}
+          remote_key: ${{ secrets.NIGHTLIES_RSYNC_KEY }}
+          remote_host_key: ${{ secrets.NIGHTLIES_RSYNC_HOST_KEY }}
+      - shell: bash
+        name: Show local repo sync from remote
+        run: |
+          for i in `ls -t repo/org/apache/arrow`; do
+            echo "- $i: $(find repo/org/apache/arrow/$i -mindepth 1 -maxdepth 1 -type d \
+            | wc -l \
+            | xargs) versions available"
```

Comment:
Is this `| xargs` needed?

Reply:
`xarg` in this case help us with one-line string trimming report about number of version available:

```
- flight-sql:        3 versions available
- arrow-vector:        7 versions available
- flight-grpc:        4 versions available
```

Reply:
I see. Thanks.